src/main/webapp/index.html

```diff
@@ -1,43 +0,0 @@
-<!-- Copyright 2020 Google LLC
```

Comment:
Are we supposed to delete this file?

Reply:
it was creating conflict with master and was not needed in this branch